chart/values.yaml

```diff
@@ -371,7 +371,7 @@ features:
       resource_directory_key: ~
 
   # Configuration for the external database; see also features.embedded_database.  Please refer to
-  # https://kubecf.suse.dev/docs/getting-started/kubernetes-deploy/#external-database for details.
+  # https://kubecf.io/docs/getting-started/kubernetes-deploy/#external-database for details.
```

Comment:
This should go to https://kubecf.io/docs/deployment/kubernetes-deploy/#external-database probably

Reply:
Yep, fixed.